src/composables/trade/useSor.spec.ts

```diff
@@ -81,7 +80,7 @@ describe('setSwapCost', () => {
     'sg'
   );
 
-  const mockedSorManager = mocked(sorManager);
+  const mockedSorManager = jest.mocked(sorManager);
```

Comment:
See https://github.com/facebook/jest/pull/12089
Using mocked from utils is deprecated